### DISCUSSION TITLE
I get "services.pgadmin.depends_on" must be a list on deploy.

### DISCUSSION THREAD

**Discussion by luanjubica-helius** [fastapi/fastapi#7116] [Questions]
Hello,
while running :

bash ./scripts/deploy.sh (with the env variables set) I get two things : 

>/usr/local/lib/python3.8/dist-packages/docker_auto_labels/docker_auto_labels.py:16: YAMLLoadWarning: calling yaml.load() ?>without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
> return yaml.load(docker_file)

which seems a warning after all, and : 
>services.pgadmin.depends_on must be a list

Which seems blocking.

Please advise.

**Comment by Mause** [comment]
Have you opened this issue in the correct repository? There is no such script in this one

**Comment by luanjubica-helius** [comment]
Right, I apologize on the other repo

**Comment by tiangolo** [comment]
Thanks for the help here @Mause ! :clap: :bow: 

Thanks for reporting back and closing the issue @luanjubica-helius  :+1:
